src/js/components/MenuComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react/addons";
+
+import Util from "../helpers/Util";
+
+import MenuItemComponent from "../components/MenuItemComponent";
+
+var MenuComponent = React.createClass({
+  "displayName": "MenuComponent",
+
+  propTypes: {
+    children: Util.getComponentTypeValidator(MenuItemComponent),
```

Comment:
Can we use the existing `React.PropTypes.instanceof(MenuItemComponent)`? https://facebook.github.io/react/docs/reusable-components.html